wp-content/plugins/wporg-learn/inc/post-meta.php

```diff
@@ -257,32 +257,32 @@ function get_workshop_duration( WP_Post $workshop, $format = 'raw' ) {
 }
 
 /**
- * Get a list of locales that are associated with at least one workshop.
- *
- * Optionally only published workshops.
+ * Get a list of locales that are associated with at least one post of the specified type.
  *
  * @param string $meta_key
+ * @param string $post_type
  * @param string $label_language
- * @param bool   $published_only
+ * @param string $post_status
  *
  * @return array
  */
-function get_available_workshop_locales( $meta_key, $label_language = 'english', $published_only = true ) {
+function get_available_post_type_locales( $meta_key, $post_type, $label_language = 'english', $post_status ) {
```

Comment:
I get this notice on PHP `8.0.28`. We don't run that on prod yet, but will in the near future, so it'd be good to be prepared now IMO.

`PHP Deprecated:  Required parameter $post_status follows optional parameter $label_language in wp-content/plugins/wporg-learn/inc/post-meta.php on line 269`

Reply:
Fixed in https://github.com/WordPress/Learn/pull/1500/commits/4cb6519b767e474cbdbea77e3f7c8416f7909388